scripts/get

```diff
@@ -48,25 +58,37 @@ if [ -n "$PKG_URL" -a -n "$PKG_SOURCE_NAME" ]; then
     sleep 1
   done
 
-  if ! [ -f $SOURCES/$1/$PKG_SOURCE_NAME -a "$(cat $STAMP 2>/dev/null)" == "$PKG_URL" ]; then
-    rm -f $SOURCES/$1/$PKG_SOURCE_NAME $STAMP
+  if ! _get_file_already_downloaded $1; then
```

Comment:
As per irc discussion, removed one more level of indentation. I've also updated the header as this file is now a complete rewrite. :)